src/components/Task/Drawer/Form/Form.tsx

```diff
@@ -527,12 +597,32 @@ const TaskDrawerForm = ({
           <Divider />
           <Box m={2}>
             <Grid container spacing={1} justify="flex-end">
-              <Grid item>
+              <Grid container item xs={8} justify="flex-start">
+                <Button
+                  size="large"
+                  variant="contained"
+                  className={classes.removeButton}
+                  onClick={() => handleRemoveDialog(true)}
+                >
+                  {deleting ? (
+                    <>
+                      <CircularProgress color="primary" size={20} />
+                      &nbsp;
```

Comment:
This intentional?

Reply:
I think I added it, but it's probably not needed. I'll delete it.

Reply:
To be clear I was commenting on the `&nbsp;` line only, not the progress indicator. Looks like you removed the whole indicator...

Reply:
Ahhhh... gotcha haha. I was a bit confused as the dialog component is focused on the screen so the user doesn't really see this loading indicator but I'll add it back. I think I just copied the ```&nbsp;``` that was in the other loading indicator on the form. Does this need to be there? https://github.com/CruGlobal/mpdx-react/blob/811726cf039830f5449da17790f2ec28f6c6aa7b/src/components/Task/Drawer/Form/Form.tsx#L543-L547

Reply:
Oh weird that it's hidden. Maybe there's a better place for it or it should show in the modal before it closes.

`&nbsp;` (non breaking space) is just a space that won't collapse into other spaces. Seems odd that you'd need a space character around a loading indicator. Usually it's a hack to get extra spacing or a way to HTML encode things that need the spacing.

Reply:
It's not necessarily hidden, just not what the user is focused on.
<img width="680" alt="Screen Shot 2021-03-25 at 9 23 19 AM" src="https://user-images.githubusercontent.com/39680460/112479921-df19b680-8d4b-11eb-91b5-817040f7fca7.png">

 I'm going to move it into the modal and switch out the text for the loading indicator while it's loading.